test/e2e/resource_generators.go

```diff
@@ -69,16 +70,51 @@ type azureConfig struct {
 }
 
 var (
-	// TODO Parameterize some of these variables
-	location       = GetRegion()
-	vmSize         = "Standard_D2s_v3"
-	namespace      = "default"
-	imageOffer     = "capi"
-	imagePublisher = "cncf-upstream"
-	imageSKU       = "k8s-1dot16dot6-ubuntu-1804"
-	imageVersion   = "latest"
+	LOCATION        string
+	VM_SIZE         string
+	NAMESPACE       string
+	IMAGE_OFFER     string
+	IMAGE_PUBLISHER string
+	IMAGE_SKU       string
+	IMAGE_VERSION   string
 )
 
+func (c *ClusterGenerator) VariablesInit() {
+
+	v := viper.New()
+
+	v.SetDefault("LOCATION", GetRegion())
+	v.SetDefault("VM_SIZE", "Standard_D2s_v3")
+	v.SetDefault("NAMESPACE", "default")
+	v.SetDefault("IMAGE_OFFER", "capi")
+	v.SetDefault("IMAGE_PUBLISHER", "cncf-upstream")
+	v.SetDefault("IMAGE_SKU", "k8s-1dot16dot6-ubuntu-1804")
```

Comment:
let's remove the image defaults since we handle the image defaults in the code now to choose the right image based on the k8s version

Reply:
ok, remove all `imageSKU`, `imagePublisher`, `imageOffer`, `imageVersion`? where that come from? did not find in the e2e code